### ISSUE TITLE
All calibration buttons are grayed out

### ISSUE THREAD

**Issue by smwdscada** [bambulab/BambuStudio#5474]
### Bambu Studio Version

1.10.1.50

### Where is the application from?

Bambu Lab Official website

### OS version

Windows 10

### Printer

Bambu X1 Carbon

### How to reproduce

When I go to the calibration tab and select my printer, all the calibration buttons for Flow Dynamics and Flow Rate are grayed out.  My printer is online and connected.  I have performed these calibrations in the past so at some point they used to work.
![calib1](https://github.com/user-attachments/assets/49739461-5a18-47d6-8b73-c337c027c9bf)
![calib2](https://github.com/user-attachments/assets/83956dde-abb3-417a-aa53-930a5d1a6d09)

### Actual results

Nothing happens when I click on the calibration buttons.

### Expected results

I expect to see the calibration wizard that walks the user through the process.

### Project file & Debug log uploads

[Bambu.zip](https://github.com/user-attachments/files/18027661/Bambu.zip)

### Checklist of files to include

- [X] Log file
- [X] Project file

**Comment by hotellonely** [comment]
Yes it's still presenting. the only workaround is to open the tab as soon as you launch bambu studio.

**Comment by SaltyAsFrig** [comment]
Same issue here too (Windows version). Can also confirm that hotellonely's workaround works too.

**Comment by maruel** [comment]
See discussion at https://www.reddit.com/r/BambuLab/comments/1gpqypp.

**Comment by R-A-S** [comment]
It has the same issue on Mac OS with the latest version of BambuStudio. It seems like all platforms have this issue.

**Comment by zhimin-zeng-bambulab** [comment]
Thanks for reporting this issue, we  will fix it in the next version. You can reconnect the printer on the device page to update the status.

**Comment by zhimin-zeng-bambulab** [comment]
It has been fixed in the latest version. You can have a try.
